CHANGELOG.md

```diff
@@ -27,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   during reconnection.
 - Update `startVideoPreviewForVideoInput` to support filters in the preview window.
 - Update browser demo to showcase preview filter capability.
+- Move `toLowerCasePropertyNames` to a static method inside utils.js.
```

Comment:
Few suggestions:
- I think we can club these two changelogs together to learn about the scope of the change in one place. 
- It is not "a static method" right? 
- We do not have a `util.js`, so we change it to `Utils.ts`.